lntest/itest/lnd_misc_test.go

```diff
@@ -532,41 +532,35 @@ func testGarbageCollectLinkNodes(ht *lntemp.HarnessTest) {
 // testRejectHTLC tests that a node can be created with the flag --rejecthtlc.
 // This means that the node will reject all forwarded HTLCs but can still
 // accept direct HTLCs as well as send HTLCs.
-func testRejectHTLC(net *lntest.NetworkHarness, t *harnessTest) {
+func testRejectHTLC(ht *lntemp.HarnessTest) {
 	//             RejectHTLC
 	// Alice ------> Carol ------> Bob
 	//
 	const chanAmt = btcutil.Amount(1000000)
-	ctxb := context.Background()
+	alice, bob := ht.Alice, ht.Bob
 
 	// Create Carol with reject htlc flag.
-	carol := net.NewNode(t.t, "Carol", []string{"--rejecthtlc"})
-	defer shutdownAndAssert(net, t, carol)
+	carol := ht.NewNode("Carol", []string{"--rejecthtlc"})
```

Comment:
what is the reason that we no longer need to defer a clean up of new nodes? (probs a noob question, sorry 🙈  still getting familiar with the new structure)

Reply:
so the main reason is [here](https://github.com/lightningnetwork/lnd/blob/master/lntemp/harness.go#L317). In short we now take advantage of the `testing`'s`t.Cleanup`, which will run functions in the end, kinda like `defer`.(and questions are always welcomed!🥳)